### PR TITLE
Fix: Remove details about SelectorSpread plugin

### LINKED ISSUE #42676: Pod Topology Spread Constraints: SelectorSpread plugin was removed
> https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/#cluster-level-default-constraints
> Note: The SelectorSpread plugin is disabled by default. The Kubernetes project recommends using PodTopologySpread to achieve similar behavior.

According to [this page section](https://kubernetes.io/docs/reference/scheduling/config/#scheduler-configuration-migrations) (check the "v1beta3 → v1" tab) "The scheduler plugin SelectorSpread is removed, instead, use the PodTopologySpread plugin (enabled by default) to achieve similar behavior." But [the page](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/#cluster-level-default-constraints) still mentions the SelectorSpread plugin.

Also this plugin is mentioned in [another page section](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/#internal-default-constraints).

### PR DESCRIPTION
**Pull request by tamilselvan1102**
Issue #42676

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@tamilselvan1102 I see the bot added ['do-not-merge/invalid-commit-message'](https://github.com/kubernetes/website/pull/42759#issuecomment-1695511119) due to the commit message containing `fixes #<issue number>` ([Blocked Keywords Guide](https://www.kubernetes.dev/docs/guide/pull-requests/#do-not-use-github-keywords-or-mentions-within-your-commit-message)). Could you please modify the commit message to help progress this PR? We prefer to exclude that aspect.

**Comment by aj11anuj** [comment]
Ahh, @tamilselvan1102 maybe you forgot to change commit message of this PR, that's why its not getting reviewed till then.

**Review by kerthcet — COMMENTED** [review]
I think we can remove the content of `selectorSpread` as already removed and we have mentioned this in **scheduling configuration migrations**. so
/lgtm

File: content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 1]
```diff
@@ -527,9 +527,7 @@ profiles:
 ```
 
 {{< note >}}
-The [`SelectorSpread` plugin](/docs/reference/scheduling/config/#scheduling-plugins)
-is disabled by default. The Kubernetes project recommends using `PodTopologySpread`
-to achieve similar behavior.
+The 'PodTopologySpread' plugin is enabled by default.
```
Comment (kerthcet) [inline content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:530]: I think we can remove this. It's well known.
/lgtm cancel

Comment (tengqm) [reply content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:530]: @tamilselvan1102 Still working on this?

Comment (tamilselvan1102) [reply content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:530]: Updated as per @kerthcet  comment and waiting for approval

Comment (tengqm) [reply content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:530]: @tamilselvan1102 Forgot to `git commit --amend` or `git push -f`?

Comment (tamilselvan1102) [reply content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:530]: Changes are reflecting in the preview page 
https://deploy-preview-42759--kubernetes-io-main-staging.netlify.app/docs/concepts/scheduling-eviction/topology-spread-constraints/#cluster-level-default-constraints

Comment (tengqm) [reply content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:530]: @tamilselvan1102 The suggestion was to remove the note completely, and it seemed that you agreed to this suggestion but you didn't "updated as per @kerthcet comment".

Comment (tamilselvan1102) [reply content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md]: I was misunderstood. now removed. Please confirm the changes

**Comment by tengqm** [comment]
This is ready to go provided the following two things are done:

- squash the PR into a single commit;
- fix the PR message (the one you commit to your clone), you cannot reference an issue number there.

**Comment by tamilselvan1102** [comment]
@tengqm @kerthcet 
Closed this PR because squash went wrong. Created new one, please validate it.
Sorry for the inconvenient cause.
https://github.com/kubernetes/website/pull/43216

**Pull request closed without merging**